modules/openapi/tests/src/OperationSpec.scala

```diff
@@ -19,649 +19,585 @@ final class OperationSpec extends munit.FunSuite {
+                            |
+                            |use smithytranslate#contentType
+                            |
+                            |service FooService {
+                            |    operations: [
+                            |        TestOperationId
+                            |    ]
+                            |}
+                            |
+                            |@http(
+                            |    method: "GET",
+                            |    uri: "/test",
+                            |    code: 200,
+                            |)
+                            |operation TestOperationId {
+                            |    input: Unit,
+                            |    output: Okay,
+                            |}
+                            |
+                            |structure Okay {
+                            |    @httpPayload
+                            |    @required
+                            |    @contentType("application/json")
+                            |    body: Body,
+                            |}
+                            |
+                            |structure Body {
+                            |    @required
+                            |    s: String,
+                            |}
+                            |""".stripMargin
 
     TestUtils.runConversionTest(openapiString, expectedString)
   }
 
-  test("operation - request reference") {
-    val openapiString = """|openapi: '3.0.'
-                   |info:
-                   |  title: test
-                   |  version: '1.0'
-                   |paths:
-                   |  /test:
-                   |    post:
-                   |      operationId: testOperationId
-                   |      requestBody:
-                   |        $ref: '#/components/requestBodies/generic'
-                   |      responses:
-                   |        '200':
-                   |          $ref: '#/components/responses/okay'
-                   |components:
-                   |  requestBodies:
-                   |    generic:
-                   |      required: true
-                   |      content:
-                   |        application/json:
-                   |          schema:
-                   |            type: object
-                   |            properties:
-                   |              s:
-                   |                type: string
-                   |            required:
-                   |              - s
-                   |  responses:
-                   |    okay:
-                   |      content:
-                   |        application/json:
-                   |          schema:
-                   |            type: object
-                   |            properties:
-                   |              sNum:
-                   |                type: integer
-                   |""".stripMargin
+  test("operation - response reference 204") {
```

Comment:
Kinda confused why scalafmt went to town on this file.. but this is the new test, the rest are unchanged